src/components/Vote.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react"
+import * as api from "../api"
+
+export default function Vote({ votes, id }) {
+	const [update, setUpdate] = useState(0)
+	const [errorMsg, setErrorMsg] = useState()
+
+	const downVote = () => {
+		setUpdate((current) => {
+			return current - 1
+		})
+		api.patchReview(id, {
+			inc_votes: -1,
+		}).catch((err) => {
+			setErrorMsg(err.message)
+		})
+	}
+
+	const upVote = () => {
+		setUpdate((current) => {
+			return current + 1
+		})
+		api.patchReview(id, {
+			inc_votes: 1,
+		}).catch((err) => {
+			setErrorMsg(err.message)
+		})
```

Comment:
At the moment I can upVote or downVote as many times as I like. Is this what you'd like a user to be able to do as opposed to only voting once either way?